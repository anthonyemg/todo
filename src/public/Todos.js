import React from 'react';

class Todos extends React.Component {
  constructor(props) {
    super(props);
  }
  render() {
    return <div>Todos</div>;
  }
}

export default Todos;
