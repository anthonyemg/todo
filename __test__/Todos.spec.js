import React from 'react';
import renderer from 'react-test-renderer';
import Todos from '../src/public/Todos';

test('Todos renders correctly', () => {
  const component = renderer.create(<Todos />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
