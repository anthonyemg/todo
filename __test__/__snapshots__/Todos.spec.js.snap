// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todos renders correctly 1`] = `
<div>
  Todos
</div>
`;
